Sources/FoundationEssentials/FileManager/SearchPaths/FileManager+WindowsSearchPaths.swift

```diff
@@ -26,6 +26,9 @@ private func _url(for id: KNOWNFOLDERID) -> URL {
 
 func _WindowsSearchPathURL(for directory: FileManager.SearchPathDirectory, in domain: FileManager.SearchPathDomainMask) -> URL? {
     switch (directory, domain) {
+    case (.autosavedInformationDirectory, _):
```

Comment:
Oh, good question. I did try to model this after the XDG path.

Reply:
I think this is likely a bug in the XDG implementation. Since the directory is within a user-specific directory it'd make sense that it's only available within the user domain mask. I also see that on Darwin, the same directory is indeed only available within the user domain mask. IMO I think we should update the XDG implementation and this one to make them both only match the user domain mask. Do you think that sounds reasonable? Happy to update the XDG one in a separate PR if you want to keep this one limited to the Windows changes

Reply:
Yes, I'd rather split the two implementations, but I don't see any concern with the change.

Reply:
https://github.com/apple/swift-foundation/pull/637 should address the XDG path.

Reply:
Awesome, thanks!